Coop_Listing_Site/Coop_Listing_Site/Views/Coop/AddOpportunity.cshtml

```diff
@@ -166,13 +166,34 @@
         <div class="form-group">
             @Html.LabelFor(model => model.Department, htmlAttributes: new { @class = "control-label col-md-2" })
             <div class="col-md-10">
-                @Html.DropDownList("DepartmentIDs", "Select One (Or none)...")
+                @Html.DropDownList("DepartmentIDs", "Select One")
             </div>
         </div>
         <div class="form-group">
-            @Html.Label("Major", htmlAttributes: new { @class = "control-label col-md-2" })
+            @Html.Label("Majors", htmlAttributes: new { @class = "control-label col-md-2" })
             <div class="col-md-10">
-                @Html.DropDownList("MajorIDs", "Select One")
+                <select id="Majors" on>
+                    <option id="top-opt" value>Choose a Department first</option>
```

Comment:
@Kethsar  
Subject of reference: <option id="top-opt" value> 
Question in regards: Why is value not instantiated? Is this satisfied through the selection of the option?